### ISSUE TITLE
After upgrading to 0.67.1 debug nodes / inject are greyed out, HA nodes shows no status

### ISSUE THREAD

**Issue by HaraldGithub** [hassio-addons/addon-node-red#1938]
# Problem/Motivation

Running latest version of NR within HA as and add-on I decided to upgrade node-red-websocket to latest which is 0.67.1 and after that I've been prompted to upgrade HA nodes and I did but realized that all of my debug nodes and inject nodes are grayed out, nothing displays inside debug window nor any HA node shows connection status. Nodes work as intended, but there's no feedback is provided by Node-Red.

Maybe it's the same issue, which is posted here:

https://github.com/zachowj/node-red-contrib-home-assistant-websocket/issues/1480

**Comment by tomadan1968** [comment]
> # Problem/Motivation
> Running latest version of NR within HA as and add-on I decided to upgrade node-red-websocket to latest which is 0.67.1 and after that I've been prompted to upgrade HA nodes and I did but realized that all of my debug nodes and inject nodes are grayed out, nothing displays inside debug window nor any HA node shows connection status. Nodes work as intended, but there's no feedback is provided by Node-Red.
> 
> Maybe it's the same issue, which is posted here:
> 
> [zachowj/node-red-contrib-home-assistant-websocket#1480](https://github.com/zachowj/node-red-contrib-home-assistant-websocket/issues

Hello, the same problem for me....

**Comment by Tykwondo1** [comment]
same problem

**Comment by sanderlv** [comment]
Same here, ip:1880 no issue

**Comment by jamesahendry** [comment]
I am the same - ip:1800 no issue or go into the inject node and "Inject Now"

**Comment by HaraldGithub** [comment]
> I am the same - ip:1800 no issue or go into the inject node and "Inject Now"

Maybe a workaround, but no solution!
And your "workaround" dosen't fix the issue with no displaying of the node status.

**Comment by TETZUO** [comment]
Same issue here, the flows are working but no status is displayed
![image](https://github.com/user-attachments/assets/ec6bed30-2b93-464a-b111-398703658922)


Inject timestap nodes also do not work I just get a 4 pointed move arrow with no option to click

**Comment by echopage1964** [comment]
I have IP 1880, and I have the same issue

**Comment by AndLindemann** [comment]
Same issue. After upgrade from 18.0.5 to 18.1.1, no status is shown on any nodes and actions can no longer be selected / other drop-downs that rely on HA also don‘t provide any selections anymore. Rolled back to 18.0.5 and everything is good again.

**Comment by Markus730** [comment]
I Have the same Problem:
In the Web Interface the Status of the Nodes after 45 Minutes.
And high 40% Nodred Cpu usage
Starting with this Versions:
HA: 2024.8
Node red 18.0.5
Node-RED Companion Integration v4.02
node-red-contrib-home-assistant-websocket 0.67.2

**Comment by Tykwondo1** [comment]
Same issue. After upgrade from 18.0.5 to 18.1.1,Rolled back to 18.0.5

**Comment by mrtramplefoot** [comment]
Same issue here on 18.1.1

**Comment by pro2call-nl** [comment]
same issue on 18.1.1. reverted back to 18.0.5

**Comment by MegaTheLEGEND** [comment]
Same issue. Version 18.1.1

**Comment by sanderlv** [comment]
I have the same issue on my HA
Core 2024.10.0
Supervisor 2024.09.1
Operating System 13.1
Frontend 20241002.2
NR Addon: 18.1.1
node-red-contrib-home-assistant-websocket 0.73.0
--> where going to ip:1880 shows the status

But I do not have this issue on my test instance of HA:
Core 2024.10.0
Supervisor 2024.09.1
Operating System 13.1
Frontend 20241002.2
NR Addon: 18.1.1
node-red-contrib-home-assistant-websocket 0.73.0

Where to start this troubleshooting? Wheres the difference?

**Comment by yuckypants** [comment]
Is no one looking at this? This is still occurring and new tickets are just being closed and referenced here. But I don't see any devs working on it...

**Comment by 3ative** [comment]
> Is no one looking at this? This is still occurring and new tickets are just being closed and referenced here. But I don't see any devs working on it...

The DEVs ARE aware and it's being looked in to.

**Comment by 3ative** [comment]
It weird that so many have this issue. It's all working fine here:
https://github.com/user-attachments/assets/df15f9b3-9147-4a46-9eb1-9b7fe95a9eb4

**Comment by sanderlv** [comment]
> It weird that so many have this issue. It's all working fine here
> I made this short video this morning...
> https://github.com/user-attachments/assets/df15f9b3-9147-4a46-9eb1-9b7fe95a9eb4
> 
> 

Why is that weird? I have 2 HA Instances. 1 is ok, 1 has it (see above).

**Comment by tomadan1968** [comment]
Yes, there are problems, probably not everyone has them but they exist and we can no longer make automations in node red by accessing the addon in home assistant, it does not show under each node the date and time when they were triggered, if you want to create an automation no it also shows the entities in the home assistant so that you can select them and the inconvenience if you are not at home to access it in the browser so that you can add an automation or modify... and these problems have appeared for more than a month.

**Comment by yuckypants** [comment]
> Yes, there are problems, probably not everyone has them but they exist and we can no longer make automations in node red by accessing the addon in home assistant, it does not show under each node the date and time when they were triggered, if you want to create an automation no it also shows the entities in the home assistant so that you can select them and the inconvenience if you are not at home to access it in the browser so that you can add an automation or modify... and these problems have appeared for more than a month.

Not only have they existed for over a month, but I have yet to see a dev comment that it's being worked. All I'm seeing are users, possibly moderators?, keep closing duplicate reports.

**Comment by cannos88** [comment]
same problem

**Comment by Phoeniix** [comment]
Same problem here.

**Comment by TETZUO** [comment]
My instance does not have that issue. I did find after leaving it open for a good 30 minutes it started to work.

Could be an issue with the number of entities in home assistant.
________________________________

**Comment by jamesahendry** [comment]
Mine works in firefox http://ipaddress:1880 but not within the HA addon.  I've reduced the number of nodes with issues down to 13 but its impossible to reduce them to zero because some of my devices are unavailable at certain times by design.

**Comment by michaelblight** [comment]
I think the problem is more fundamental. I use NR in a separate instance, but have the addon installed. Therefore I can start from scratch. I deleted the addon (and deleted the flow files folder because it doesn't get cleaned up), then reinstalled.

Test 1: Brand new install of NR addon
- Dragged on an inject node and deployed (ignoring unused config nodes warning since the HA config is initially there but unused). (I may have also restarted the NR addon, can't remember).
- Result: Cannot click to start inject node (ie. broken)

Test 2: Remove all HA nodes and retry
- From the above result, I deleted the HA config node and redeployed.
- Result: No change, still cannot click to start the inject node.
- Restarted the NR addon
- Result: All good, can click to start the inject node.

Test 3: Put some HA nodes back and retry
- From the above working state
- Dragged on a current-state node and created a new HA config node for it with "Using the Home Assistant Add-on", then deployed
- Result: Can click on the inject node and it says ("injected"), however attaching a debug to it does not result in output, so it's now broken.
- Restarted the NR addon
- Result: Cannot click to start inject node (ie. broken)
- Deleted the current-state and config nodes, deployed and restarted the NR addon
- Result: Working again (until you add an HA node).

A brand new install of the addon currently comes with v0.73.0 of the HA nodes. Upgrading to v0.74.1 makes no difference. Setting the log level to debug shows nothing relevant.

As others have observed, accessing via ip:1880 works in all cases.

**Comment by blaxxor** [comment]
> What fixed this for me was fixing node issues. You can see where these are found in the UI in this comment: [node-red/node-red#4902 (comment)](https://github.com/node-red/node-red/issues/4902#issuecomment-2393137430)
> 
> As soon as everything was fixed I was able to interact with inject nodes and such again.

Didn't work for mer. I hade 5 issues. Fixed them all but still can't get it running..

**Comment by m3ki** [comment]
It seems this issue is now fixed, at least for me
**Hass:**
Core 2024.10.4
Supervisor 2024.10.3
Operating System 13.2 
**Nodered**
Current version: **18.1.1**
node-red-contrib-home-assistant-websocket: **0.74.2**

**Comment by EdGalleJr** [comment]
Same problem.

**Comment by MegaTheLEGEND** [comment]
It works for me after connecting to the 1880 port (I could not get the authentication to work so I very temporarily turned in 'leave the front door open') once in, I fixed the home assistant node errors and now it works in the home assistant web interface.

**Comment by atomic10** [comment]
Solution for this fault is to update the HA websocket in node red palette manager 
[](https://github.com/hassio-addons/addon-node-red/issues/1971#issuecomment-2464276252)

**Comment by feelitloveit** [comment]
> Solution for this fault is to update the HA websocket in node red palette manager
> 
> [#1971 (comment)](https://github.com/hassio-addons/addon-node-red/issues/1971#issuecomment-2464276252)

This worked for me!

**Comment by frenck** [comment]
Fixed by https://github.com/hassio-addons/addon-node-red/releases/tag/v19.0.0
